community/csv/src/main/java/org/neo4j/csv/reader/BufferedCharSeeker.java

```diff
@@ -267,9 +267,9 @@ private static boolean getTrimStringIgnoreErrors( Configuration config )
     }
 
     @Override
-    public <EXTRACTOR extends Extractor<?>> EXTRACTOR extract( Mark mark, EXTRACTOR extractor )
+    public <EXTRACTOR extends Extractor<?>> EXTRACTOR extract( Mark mark, EXTRACTOR extractor, String optionalData )
```

Comment:
Maybe we could have an overloaded version without the optional parameter that calls this one with `null` to avoid a lot of additional null arguments in the code below.